doc/changelogs/CHANGELOG_V7.md

```diff
@@ -21,6 +22,125 @@
+* [[`ccc6e75bea`](https://github.com/nodejs/node/commit/ccc6e75bea)] - **test**: ensure nextTick is not scheduled in exit (Jeremiah Senkpiel) [#9555](https://github.com/nodejs/node/pull/9555)
+* [[`00a5490ecd`](https://github.com/nodejs/node/commit/00a5490ecd)] - **test**: increase coverage of process.emitWarning (Jeremiah Senkpiel) [#9556](https://github.com/nodejs/node/pull/9556)
+* [[`f3db5e4720`](https://github.com/nodejs/node/commit/f3db5e4720)] - **test**: refactor test-zlib.js (Rich Trott) [#9544](https://github.com/nodejs/node/pull/9544)
+* [[`58fc7a137c`](https://github.com/nodejs/node/commit/58fc7a137c)] - **test**: change from setTimeout to setImmediate (MURAKAMI Masahiko) [#9578](https://github.com/nodejs/node/pull/9578)
+* [[`e7eb9ccdcf`](https://github.com/nodejs/node/commit/e7eb9ccdcf)] - **test**: improve test-stream2-objects.js (Yoshiya Hinosawa) [#9565](https://github.com/nodejs/node/pull/9565)
+* [[`dae3d3e53c`](https://github.com/nodejs/node/commit/dae3d3e53c)] - **test**: refactor test-next-tick-error-spin (Rich Trott) [#9537](https://github.com/nodejs/node/pull/9537)
+* [[`8c859d58ab`](https://github.com/nodejs/node/commit/8c859d58ab)] - **test**: refactor test-tls-inception (Rich Trott) [#9536](https://github.com/nodejs/node/pull/9536)
+* [[`96471556b5`](https://github.com/nodejs/node/commit/96471556b5)] - **test**: move tick-processor tests to own directory (Rich Trott) [#9506](https://github.com/nodejs/node/pull/9506)
+* [[`342d8e05cb`](https://github.com/nodejs/node/commit/342d8e05cb)] - **test**: refactor inspector-helper.js (Rich Trott) [#9499](https://github.com/nodejs/node/pull/9499)
+* [[`dab3e451ac`](https://github.com/nodejs/node/commit/dab3e451ac)] - **test**: refactor make-callback-recurse test (Rich Trott) [#9498](https://github.com/nodejs/node/pull/9498)
+* [[`2a9625656d`](https://github.com/nodejs/node/commit/2a9625656d)] - **test**: move timer-dependent test to sequential (Rich Trott) [#9487](https://github.com/nodejs/node/pull/9487)
+* [[`ee7606940a`](https://github.com/nodejs/node/commit/ee7606940a)] - **test**: fix helper-debugger-repl.js (Rich Trott) [#9486](https://github.com/nodejs/node/pull/9486)
+* [[`9491352b86`](https://github.com/nodejs/node/commit/9491352b86)] - **test**: remove watchdog in test-debug-signal-cluster (Rich Trott) [#9476](https://github.com/nodejs/node/pull/9476)
+* [[`6a94ffb1cf`](https://github.com/nodejs/node/commit/6a94ffb1cf)] - **test**: output tap13 instead of almost-tap (Johan Bergström) [#9262](https://github.com/nodejs/node/pull/9262)
+* [[`02c2bf7d34`](https://github.com/nodejs/node/commit/02c2bf7d34)] - **timers**: use consistent checks for canceled timers (Jeremiah Senkpiel) [#9685](https://github.com/nodejs/node/pull/9685)
+* [[`ee65b4872d`](https://github.com/nodejs/node/commit/ee65b4872d)] - **tls**: fix leak of WriteWrap+TLSWrap combination (Fedor Indutny) [#9586](https://github.com/nodejs/node/pull/9586)
```

Comment:
Judging from the conversation on https://github.com/nodejs/node/pull/9626 (v4.x backport of the same commit) this one is probably “notable changes”-worthy